src/index.html

```diff
@@ -5,9 +5,395 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="shortcut icon"
+      href="./images/favicon.png"
+      type="image/x-icon"
+    >
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__container container">
+        <div class="header__top">
+          <div class="header__phone">
+            <a href="tel:+1 234 555-55-55" class="header__link">+1 234 555-55-55</a>
+            <a href="tel:+1 234 555-55-55" class="header__phone-img">
+              <img src="./images/Phone.svg" alt="Phone">
+            </a>
+          </div>
+          <a href="#">
+            <img
+              src="./images/logo.png"
+              alt="Miami Condo Kings logo"
+              class="logo header__logo"
+            >
+          </a>
+          <div class="header__call">
+            <a href="tel:+1 234 555-55-55" class="header__link">call to order</a>
+          </div>
+        </div>
+        <div class="rectangle"></div>
+        <div>
+          <h1 class="header__title">Miami<br> Condo Kings</h1>
+        </div>
+
+        <nav class="header_nav">
+
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link">Home</a>
+            </li>
+            <li class="nav__item">
+              <a href="#about" class="nav__link">About Us</a>
+            </li>
+            <li class="nav__item">
+              <a href="#gallery" class="nav__link">Gallery</a>
+            </li>
+            <li class="nav__item">
+              <a href="#consultation" class="nav__link">Consultation</a>
+            </li>
+            <li class="nav__item">
+              <a href="#testimonials" class="nav__link">Testimonials</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contact" class="nav__link">Contacts</a>
+            </li>
+          </ul>
+        </nav>
+
+        <div class="burger">
+          <span class="burger__icon"></span>
+        </div>
+        <div class="burger__menu">
+          <div class="burger__container">
+            <div class="burger__logo-small">
+              <a href="#">
+                <img
+                    src="./images/burger_logo.png"
+                    alt="Miami Condo Kings logo"
+                    class="burger__logo burger__header-logo"
+                  >
```

Comment:
щось тут відступи завеликі
```suggestion
                <img
                  src="./images/burger_logo.png"
                  alt="Miami Condo Kings logo"
                  class="burger__logo burger__header-logo"
                >
```